lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/participants/hover/MavenHoverParticipant.java

```diff
@@ -112,6 +117,57 @@ public Hover onTag(IHoverRequest request) throws Exception {
 		return null;
 	}
 
+	private Hover collectWorkspaceArtifactDescription(IHoverRequest request) {
```

Comment:
As this is supposed to return only one value and is not alerating the input parameter, we'd rather call it `getWorkspaceArtifactDescription` or `computeWorkspaceArtifactDescription` or `findWorkspaceArtifactDescription`.

Reply:
OK